index.html

```diff
@@ -100,16 +130,21 @@ <h3>Multi-Post Stories <br>Gain+Glory</h3>
           </ul>
           <button class="button" type="button">See Project</button></span>
       </article  class="about-me-background">
+    </div>
+      </div>
     </section>
     <!-- about me section -->
     <section class="about-me-page">
+      <article class="about-container">
     <h2 class="titles-about-me">About me</h2>
     <p class="paragraph">
       I can help you build a product , feature or website Look through
       some of my work and experience! If you like what you see and have a
       project you need coded, don’t hestiate to contact me.
     </p>
     <button class="button-resume" type="button">get my resume</button>
+  </article>
```

Comment:
- [ ] Dear @ijosue16, Your design is almost exceptional, I'm going to mention some minor facts so it becomes flawless. here You could add more padding/Margin to `about` section. as you can see on a screen size around `980px` you're about text seems to be overlapping with the background images. This affects your webpage responsiveness.

## Figma design 
![figma-about](https://user-images.githubusercontent.com/69077061/200369757-4c4483c0-c5c7-4ebf-ad5e-a3c024ad7bc2.PNG)

## Your design
![your-about](https://user-images.githubusercontent.com/69077061/200369762-65843b0a-6d0e-48e5-9ca1-ba82f37b6d2b.png)